src/plugins/discover/public/embeddable/search_embeddable.scss

```diff
@@ -0,0 +1,12 @@
+/**
+   * 1. We want the osdDocTable__container to scroll only when embedded in an embeddable panel
```

Comment:
Are you sure we want this? The doctable component will already add a scrollbar when the height of the container exceeds the height of the rows combined.